### PR TITLE
Docs fixes.

### PR DESCRIPTION
**Pull request by ypid**
* Fixed RST warning markup.
* `borg prune` only knows "--keep-within" and not "--within".
* Use HTTPS everywhere. Especially when the website already redirects to HTTPS.
* Fixed spelling.
* Make note that AES-256 is used.
* Use ``--option`` in RST to not render "--" as "–".

  ``--options`` should probably also be used for help texts which are shown on the command line help.
  https://github.com/borgbackup/borg/blob/master/borg/archiver.py

  This also looks good in the manpage.

Thanks so much for your work!

### CODE REVIEW

File: docs/internals.rst [hunk 1]
```diff
@@ -311,28 +311,27 @@ more chunks than estimated above, because 1 file is at least 1 chunk).
 
 If a remote repository is used the repo index will be allocated on the remote side.
 
-E.g. backing up a total count of 1Mi files with a total size of 1TiB.
+E.g. backing up a total count of 1MiB files with a total size of 1TiB.
```
Comment (ThomasWaldmann) [inline docs/internals.rst]: no, it was not meant as 1 MiB, but as 2^20 count files.

File: docs/usage.rst [hunk 2]
```diff
@@ -75,7 +75,7 @@ Some "yes" sayers (if set, they automatically confirm that you really want to do
     BORG_RELOCATED_REPO_ACCESS_IS_OK
         For "Warning: The repository at location ... was previously located at ..."
     BORG_CHECK_I_KNOW_WHAT_I_AM_DOING
-        For "Warning: 'check --repair' is an experimental feature that might result in data loss."
+        For "Warning: 'check ``--repair``' is an experimental feature that might result in data loss."
```
Comment (ThomasWaldmann) [inline docs/usage.rst]: maybe '``check --repair``' ?

**Comment by ThomasWaldmann** [comment]
thanks for fixing the docs! :)

**Comment by ypid** [comment]
Thanks for the review. I updated my PR and fixed my mistakes.

**Pull request merged**